src/event.rs

```diff
@@ -258,9 +260,19 @@ where
 
     canvas.add_event_listener({
         let input_handler = input_handler.clone();
+        let state = state.clone();
+        let ctx = ctx.clone();
 
         move |event: KeyUpEvent| {
-            input_handler.borrow_mut().handle_key_up(event.code());
+            if event.code() == "Space" {
```

Comment:
I think there is no need for "Space" button special case here. 
It was supposed to prevent page scroll, which make sense only in KeyDown.

Reply:
OK, good.  Will cut this.